src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleDynamicAssociationsController.cs

```diff
@@ -17,32 +17,57 @@ namespace VirtoCommerce.CatalogModule.Web.Controllers.Api
     [Authorize]
     public class CatalogModuleDynamicAssociationsController : Controller
     {
-        private readonly IDynamicAssociationSearchService _searchService;
-        private readonly IDynamicAssociationService _service;
+        private readonly IDynamicAssociationSearchService _dynamicAssociationSearchService;
+        private readonly IDynamicAssociationService _dynamicAssociationService;
         private readonly IAuthorizationService _authorizationService;
 
-        public CatalogModuleDynamicAssociationsController(IDynamicAssociationSearchService searchService, IDynamicAssociationService service, IAuthorizationService authorizationService)
+        public CatalogModuleDynamicAssociationsController(
+            IDynamicAssociationSearchService dynamicAssociationSearchService,
+            IDynamicAssociationService dynamicAssociationService,
+            IAuthorizationService authorizationService)
         {
-            _searchService = searchService;
-            _service = service;
+            _dynamicAssociationSearchService = dynamicAssociationSearchService;
+            _dynamicAssociationService = dynamicAssociationService;
             _authorizationService = authorizationService;
         }
 
         [HttpPost]
         [Route("search")]
         public async Task<ActionResult<DynamicAssociationSearchResult>> SearchAssociation([FromBody] DynamicAssociationSearchCriteria criteria)
         {
-            var result = await _searchService.SearchDynamicAssociationsAsync(criteria);
+            var authorizationResult = await _authorizationService.AuthorizeAsync(
```

Comment:
I think no need to make new lines for 3 params, all fit into one line pretty well. >3 is ok